### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by plehegar**
We're retiring our use of travis, switching to use GitHub actions intead

### CODE REVIEW

File: .github/workflows/selt_test.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+jobs:
```
Comment (yurishkuro) [inline .github/workflows/selt_test.yml]: `.github/workflows/selt_test.yml` - self, not selt?

File: test/start.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd ./test
```
Comment (SergeyKanzhelev) [inline test/start.sh]: is it started from the folder above? Maybe this:

```suggestion
cd "$(dirname "$0")"
```

File: .github/workflows/selt_test.yml (hunk 1 above)
Comment (plehegar) [reply .github/workflows/selt_test.yml]: oops, Fixed in https://github.com/w3c/trace-context/pull/455/commits/589c051800651927f8efdfddff48f148df899551

File: test/start.sh (hunk 2 above)
Comment (plehegar) [reply test/start.sh]: I removed the shell script and moved the cd command back in the yml

**Review by yurishkuro — APPROVED** [review]

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by danielkhan — APPROVED** [review]

**Pull request merged**